doofinder-for-woocommerce/includes/api/class-store-api.php

```diff
@@ -73,7 +73,11 @@ public function create_store($api_keys)
         if (is_array($api_keys)) {
             $store_payload = $this->build_store_payload($api_keys);
             $this->log->log("store_data: ");
-            $this->log->log($store_payload);
+
+            $store_payload_log = $store_payload;
+            unset($store_payload_log["options"]);
```

Comment:
why do you remove the options values?

Reply:
It is because in that node there is sensitive information (user and pass api). And we don't want there to be that type of information because it is a public path.

Reply:
I’m asking because sometimes can be helpful know the payload that we are using to create the store, so we can try to reproduce it, because the payload is not store in our side… so should be possible remove sensitive info but keep the rest?

You said it is public that means anyone can access? Or has to be logged in Wordpress?

Reply:
I mean that it is public for everyone (without login) through URL because it is inside wp-content/plugin/log/... I think it is not convenient to store such confidential data in those files

Reply:
not confidential but what about the rest? info that anyway is public? what do you think?

Reply:
everything is clear! thank you